src/graphql/mutations/start_game.ts

```diff
@@ -27,7 +34,18 @@ export default async (
     const validationResult = await validateGame(game, t);
 
     // Game is valid, perform setup
-    return setupGame(game, validationResult.gamePlayers, t);
+    game = await setupGame(game, validationResult.gamePlayers, t);
+  });
+  return await sequelizeConnection.transaction(async (t2) => {
```

Comment:
By wrapping in a second transaction, we risk the proof request failing and then being out of luck since the game already started.  I think what I need to do is go in and rewrite everything related to serializing game state for the proof and extract it from SQL.  I'm going to do that in a follow-up PR